src/openMVG/sfm/AlembicExporter.cpp

```diff
@@ -377,6 +377,10 @@ void AlembicExporter::add(const sfm::SfM_Data &sfmdata, sfm::ESfM_Data flags_par
         // If we don't export views, skip cameras without valid pose.
         continue;
       }
+      else{
+        // If there is no intrinsic/pose defined, skip camera
+        continue;
```

Comment:
@fabiencastan this is ugly and temporary... A custom alembic flag to say that the camera is not valid (with identity pose) it may be better. Or not. I don't know.